build.js

```diff
@@ -87,7 +89,9 @@ const getCustomTestAPI = (name, member, type) => {
         ? customTests.api[name].__base.replace(/\n/g, '\n  ') + '\n  '
         : '';
     const promise = testbase.includes('var promise');
-    const callback = testbase.match(/callback([(),])/g);
+    const callback =
+      importcode.match(/callback([(),])/g) ||
```

Comment:
This was later undone in https://github.com/GooborgStudios/mdn-bcd-collector/commit/ff9cbcf4901954a9b1629c8fefccfec3fad9d03d so I'll revert it here to make CI pass.